multipy/runtime/interpreter/CMakeLists.txt

```diff
@@ -12,27 +12,24 @@ include_directories(BEFORE "${PYTORCH_ROOT}/torch/include")
 include_directories(BEFORE "${PYTORCH_ROOT}/torch/include/torch/csrc/api/include/")
 SET(MULTIPY_UTILS "${CMAKE_CURRENT_SOURCE_DIR}/../../utils")
 
-# Build cpython
-SET(PYTHON_INSTALL_DIR "${INTERPRETER_DIR}/cpython")
-SET(PYTHON_INC_DIR "${PYTHON_INSTALL_DIR}/include/python3.8")
-SET(PYTHON_INC_DIR "${PYTHON_INSTALL_DIR}/include/python3.8" PARENT_SCOPE)
-SET(PYTHON_LIB "${PYTHON_INSTALL_DIR}/lib/libpython3.8.a")
-SET(PYTHON_BIN "${PYTHON_INSTALL_DIR}/bin/python3")
-include(ExternalProject)
-ExternalProject_Add(
-  cpython
-  PREFIX cpython
-  GIT_REPOSITORY https://github.com/python/cpython.git
-  GIT_TAG v3.8.6
-  UPDATE_COMMAND ""
-  PATCH_COMMAND git apply ${CMAKE_CURRENT_SOURCE_DIR}/cpython_patch.diff
-  BUILD_IN_SOURCE True
-  CONFIGURE_COMMAND PYTHON_INSTALL_DIR=${PYTHON_INSTALL_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/configure_cpython.sh
-  BUILD_COMMAND CFLAGS=-fPIC CPPFLAGS=-fPIC make -j8
-  INSTALL_COMMAND make install
-  BYPRODUCTS ${PYTHON_MODULES} ${PYTHON_LIB} ${PYTHON_BIN} ${PYTHON_INSTALL_DIR}/lib/libssl.a ${PYTHON_INSTALL_DIR}/lib/libcrypto.a
-  LOG_OUTPUT_ON_FAILURE True
-)
+find_package (Python3 COMPONENTS Interpreter Development)
+
+message(STATUS "Python3_EXECUTABLE - ${Python3_EXECUTABLE}" )
+message(STATUS "Python3_SITELIB - ${Python3_SITELIB}" )
+message(STATUS "Python3_LIBRARIES - ${Python3_LIBRARIES}" )
+
+# Find static libpython with a preference for nolto since cross compiler version
+# LTO is problematic.
+string(REPLACE ".so" ".nolto.a" Python3_STATIC_LIBRARIES ${Python3_LIBRARIES})
+if (NOT EXISTS ${Python3_STATIC_LIBRARIES})
+  string(REPLACE ".so" ".a" Python3_STATIC_LIBRARIES ${Python3_LIBRARIES})
```

Comment:
nolto seems to be specific to libpython-static from conda. Normal Python installs don't have nolto (i.e. via pyenv). The .so file shouldn't matter since dynamically linked files don't do LTO AFAIK?